WordPress/src/main/java/org/wordpress/android/ui/mysite/cards/dashboard/CardsTracker.kt

```diff
@@ -25,8 +25,9 @@ class CardsTracker @Inject constructor(
         POST("post"),
         BLOGGING_PROMPT("blogging_prompt"),
         PROMOTE_WITH_BLAZE("promote_with_blaze"),
+        PAGES("pages"),
+        ACTIVITY("activity"),
         DASHBOARD_CARD_DOMAIN("dashboard_card_domain"),
```

Comment:
np: Sorry for my `,` nitpicking. Feel free to ignore them, but another extra comma for the last item here. 😅

Reply:
Same as above.